templates/web.template.yml

```diff
@@ -285,6 +285,7 @@ run:
                 missingok
                 delaycompress
                 compress
+                sharedscripts
```

Comment:
Our nginx logrotate config uses `sharedscripts` so this might be something we missed 10 years ago 😅 

https://github.com/discourse/discourse_docker/blob/f2d50d70087394d4a470137e05106016b64a1c9e/templates/web.template.yml#L303